### PR TITLE
Episode 19 items (kRO 2022-01-19 patch)

### PR DESCRIPTION
**Pull request by Toshiro90**
* **Addressed Issue(s)**: -

* **Server Mode**: Renewal

* **Description of Pull Request**: 

Adds over 500 new items for episode 19 (released on 2022-01-19 on kRO), as well as some missing ones contained in new item boxes.
Includes item combos and item groups, as well as trading and buyingstore flags.
Most items use generic AegisName entries (aegis_###) when no name could be found in client files.

### CODE REVIEW

**Comment by Atemo** [comment]
There is a typo in item_db_etc `Location:` is ignored as it should be `Locations:`

**Comment by Toshiro90** [comment]
> There is a typo in item_db_etc `Location:` is ignored as it should be `Locations:`

Fixed.

**Review by Atemo — CHANGES_REQUESTED** [review]

File: src/map/itemdb.hpp [hunk 1]
```diff
@@ -766,6 +766,18 @@ enum e_random_item_group {
 	IG_SEALED_SCROLL,
 	IG_SQUAD_PRIZE1,
 	IG_SQUAD_PRIZE2,
+	IG_LI_Nyangvine_Box1_26,
+	IG_LI_Nyangvine_Box2_26,
+	IG_LI_Nyangvine_Box3_26,
+	IG_Enchant_Stone_Box26,
```
Comment (Lemongrass3110) [inline src/map/itemdb.hpp:772]:
```suggestion
	IG_LI_NYANGVINE_BOX1_26,
	IG_LI_NYANGVINE_BOX2_26,
	IG_LI_NYANGVINE_BOX3_26,
	IG_ENCHANT_STONE_BOX26,
```

File: src/map/script_constants.hpp [hunk 2]
```diff
@@ -5512,6 +5512,18 @@
 	export_constant(IG_SEALED_SCROLL);
 	export_constant(IG_SQUAD_PRIZE1);
 	export_constant(IG_SQUAD_PRIZE2);
+	export_constant(IG_LI_Nyangvine_Box1_26);
+	export_constant(IG_LI_Nyangvine_Box2_26);
+	export_constant(IG_LI_Nyangvine_Box3_26);
+	export_constant(IG_Enchant_Stone_Box26);
```
Comment (Lemongrass3110) [inline src/map/script_constants.hpp:5518]: export_constant(IG_LI_NYANGVINE_BOX1_26);
	export_constant(IG_LI_NYANGVINE_BOX2_26);
	export_constant(IG_LI_NYANGVINE_BOX3_26);
	export_constant(IG_ENCHANT_STONE_BOX26);

**Review by Lemongrass3110 — COMMENTED** [review]
Only checked the source part.

**Review by Atemo — APPROVED** [review]
Good work!

**Review by Everade — APPROVED** [review]

**Pull request merged**